osu.Game/Overlays/Settings/Sections/General/UpdateSettings.cs

```diff
@@ -74,10 +74,33 @@ private void load(Storage storage, OsuConfigManager config, OsuGame game)
                     Action = () => storage.PresentExternally(),
                 });
 
+                Add(new SettingsButton
+                {
+                    Text = GeneralSettingsStrings.ExportLogs,
+                    Keywords = new[] { @"bug", "report", "logs", "files" },
+                    Action = () =>
+                    {
+                        var logStorage = Logger.Storage;
+
+                        const string archive_filename = "exports/compressed-logs.zip";
+
+                        using (var outStream = storage.CreateFileSafely(archive_filename))
+                        using (var zip = ZipArchive.Create())
+                        {
+                            foreach (string? f in logStorage.GetFiles(string.Empty, "*.log"))
+                                zip.AddEntry(f, logStorage.GetStream(f), true);
+
+                            zip.SaveTo(outStream);
+                        }
+
+                        storage.PresentFileExternally(archive_filename);
```

Comment:
Should probably behave the same as other export buttons (e.g. `Export selected skin`) by showing a notification that leads to the folder.

Reply:
The button is already hidden for non-desktop platforms. I'll look at adding the notification flow.